pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -284,6 +304,28 @@ func newRunnerPodForCR(cr *litmuschaosv1alpha1.ChaosEngine, aUUID types.UID, aEx
 				WithArgumentsNew([]string{"-c", "ansible-playbook ./executor/test.yml -i /etc/ansible/hosts; exit 0"}).
 				WithEnvsNew(getChaosRunnerENV(cr, aExList)),
 		).
+		Build()
+	if err != nil {
+		return nil, err
+	}
+	return podObj, nil
+}
+
+// newExporterPodForCR defines secondary resource #2 in same namespace as CR */
+func newExporterPodForCR(cr *litmuschaosv1alpha1.ChaosEngine, aUUID types.UID) (*corev1.Pod, error) {
+	if cr == nil {
+		return nil, errors.New("nil ChaosEngine Object")
```

Comment:
Can you change the error to:- `Error while creating exporter pod, err: chaosengine got nil`